core/src/main/kotlin/cash/grammar/kotlindsl/utils/Comments.kt

```diff
@@ -55,6 +56,22 @@ public class Comments(
     }
   }
 
+  public fun getCommentTokenInBlock(ctx: KotlinParser.NamedBlockContext): List<Token> {
```

Comment:
suggestion: name the function `getCommentsInBlock`. If you really think it should have `Token`, that's ok, but at least we need an `s` for plurals :)

Reply:
This isn't a critical comment, but I'm curious why you're iterating over the tokens in reverse order?

Reply:
Good point! It was a result of a few iterations of refactoring -- rriginally, I was trying to get tokens of the left a statement. I will change this to non-reverse order.